src/components/ContentGroup/ContentGroup.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Meta, StoryFn } from "@storybook/react";
+
+import { Button } from "../Button/Button";
+
+import { ContentGroup } from "./ContentGroup";
+
+export default {
+    title: "Components/ContentGroup",
+    component: ContentGroup,
+    argTypes: {
+        handlerToggleCollapse: {
+            action: "toggle collapse",
+        },
+    },
+} as Meta<typeof ContentGroup>;
+
+const TemplateFull: StoryFn<typeof ContentGroup> = (args) => <ContentGroup {...args} />;
+
+export const BasicExample = TemplateFull.bind({});
```

Comment:
Just create one more complex example instead of four so small, include presentation of all visual features in there.

Reply:
The example does not show the `"Context info"` value, see https://62150dbccb2d77003a2a5093-vbamgbzrqs.chromatic.com/?path=/docs/components-contentgroup--docs

Please check if the example or the implementation does not work correctly.